cmd/security-agent/app/app.go

```diff
@@ -81,14 +81,14 @@ Datadog Security Agent takes care of running compliance and security checks.`,
 		},
 	}
 
-	pidfilePath string
-	confPath    string
-	flagNoColor bool
-	stopCh      chan struct{}
+	pidfilePath   string
+	confPathArray []string
+	flagNoColor   bool
+	stopCh        chan struct{}
 )
 
 func init() {
-	SecurityAgentCmd.PersistentFlags().StringVarP(&confPath, "cfgpath", "c", "", "path to directory containing datadog.yaml")
+	SecurityAgentCmd.PersistentFlags().StringArrayVarP(&confPathArray, "cfgpath", "c", make([]string, 0), "path to a yaml configuration file")
```

Comment:
nit: StringArrayVarP(&confPathArray, "cfgpath", "c", nil, "path to a yaml configuration file")